src/utils.ts

```diff
@@ -124,3 +125,11 @@ export function fixJsonQuotes(target: string): string {
      }
      return result;
 }
+
+export function save(ent: Entity | null, entName: string, timestamp: BigInt): void {
```

Comment:
why it need to get entName ?

Reply:
Becuase  ent.save() doesn't exist

Reply:
I know this need for the `entName ` is bad and very very error-prone, but I don't think there is a better way around it.

Reply:
we might missing the asserts that .save do ..
e.g 
`assert(id !== null, "Cannot save AvatarContract entity without an ID");
    assert(
      id.kind == ValueKind.STRING,
      "Cannot save AvatarContract entity with non-string ID. " +
        'Considering using .toHex() to convert the "id" to a string.'
    );`

Reply:
I guess we could manually add them, but this is obviously not a very optimal solution...